bentoml/_internal/frameworks/transformers.py

```diff
@@ -1,86 +1,41 @@
-import re
+from __future__ import annotations
+
+import os
 import typing as t
 import logging
 import importlib.util
 from typing import TYPE_CHECKING
-from importlib import import_module
 
-from simple_di import inject
-from simple_di import Provide
+import pydantic
 
 import bentoml
 from bentoml import Tag
+from bentoml.models import Model
+from bentoml.models import ModelContext
+from bentoml.models import ModelOptions
+from bentoml.exceptions import NotFound
 from bentoml.exceptions import BentoMLException
 from bentoml.exceptions import MissingDependencyException
 
 from ..types import LazyType
 from ..utils.pkg import get_pkg_version
-from .common.model_runner import BaseModelRunner
-from ..configuration.containers import BentoMLContainer
-
-logger = logging.getLogger(__name__)
 
 if TYPE_CHECKING:
+    from bentoml.types import ModelSignature
+    from bentoml.types import ModelSignatureDict
 
-    from ..models import ModelStore
     from ..external_typing import transformers as ext
```

Comment:
Since we are simplifying a lot of concepts, probably we don't need the transformers type under `external_typing` anymore. For the pipeline type I think we should just import directly from transformers and put it under `TYPE_CHECKING`.